packages/app/src/pages/components/Login.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react'
+import {redirectTo, getCookieValue, setCookie} from '../utils'
+import * as styles from '../index.module.sass'
+
+const HOME = 'http://localhost:8000'
+const SERVER_OUTH_LOGIN = 'http://localhost:3000/oauth/login'
+
+const resetUserCookies = () => {
+    const userCookies = ['user', 'access_token', 'returnUrl']
+    
+    userCookies.forEach((cookie) => {
+        setCookie(cookie, HOME, -1)
+    })
+}
+
+interface user {
+    avatarUrl: string;
+    username: string;
+}
+
+const Login = () => {
+    const [user, setUser] = useState<user | undefined>(undefined)

```

Comment:
There is no need to pass `undefined` as argument since it's the default value for a the `useState` hook.